pkg/operation/botanist/monitoring.go

```diff
@@ -305,10 +345,27 @@ func (b *Botanist) getCustomAlertingConfigs(ctx context.Context, alertingSecretK
 }
 
 func (b *Botanist) deployGrafanaCharts(role, dashboards, basicAuth, subDomain string) error {
+	defaultSecretName := "grafana-tls"
```

Comment:
Maybe expose this as a constant somewhere and reuse it here and in `pkg/operation/botanist/secrets.go` where it is generated?